src/main/python/vantiqsdk.py

```diff
@@ -1149,11 +1193,11 @@ async def start_subscriber_transport(self):
             task = asyncio.create_task(self._subscriber.connect())
             failure_reason = None
             try:
-                if self._subscriber.connected_future is not None:
+                if self._subscriber.connected_future:
                     await self._subscriber.connected_future
             except Exception as e:
                 failure_reason = e
-            if failure_reason is not None:
+            if failure_reason is None:
```

Comment:
This seems like a mistake. Should it be "if not failure_reason"?

Reply:
Well, this is all in controlled code, but it may read easier with `if not failure_reason`.  Will update.